docs/src/content/components/position.mdx

```diff
@@ -215,6 +215,12 @@ import {
 />
 
 <TypesTable>
+   <TypesProp
+    name="allowOffscreen"
+    type="boolean"
+    desc="allow rendering of positioned element offscreen, useful when open on page load"
```

Comment:
I don't understand the "useful" comment. Might benefit from an example section where recommendations like this could benefit from room for explanation.

Reply:
Will do!